<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParameterDock</class>
 <widget class="QDockWidget" name="ParameterDock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>342</width>
    <height>917</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tool Box</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QToolBox" name="toolBox">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>307</width>
         <height>960</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Page 1</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Proxy View Tools</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QCheckBox" name="Show_Trackball_CheckBox">
             <property name="text">
              <string>Show Trackball</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Main View Tools</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QCheckBox" name="Show_Image_CheckBox">
             <property name="text">
              <string>Show Image</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QCheckBox" name="flatCheckbox">
               <property name="text">
                <string>flat</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="Show_Light_CheckBox">
               <property name="text">
                <string>Show Light</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="edgeThresholdSlider">
               <property name="value">
                <number>75</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Interactive Mode</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="Main_Interact_Mode_ComboBox">
               <item>
                <property name="text">
                 <string>Static</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Tag Plane</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>Render Mode</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="Main_Render_Mode_ComboBox">
               <item>
                <property name="text">
                 <string>Without Model Transform</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>With Model Transform</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="FeatureViewGroupBox">
          <property name="title">
           <string>Feature View Tools</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QCheckBox" name="checkBox_0">
             <property name="text">
              <string>Show Source Vector Field</string>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_1">
             <property name="text">
              <string>Show Target Vector Field</string>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_2">
             <property name="text">
              <string>Show Source Curve</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_3">
             <property name="text">
              <string>Show Target Curve</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_9">
             <property name="text">
              <string>Show Overlay Curves</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_4">
             <property name="text">
              <string>Show Source Scalar Field</string>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_5">
             <property name="text">
              <string>Show Target Scalar Field</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_6">
             <property name="text">
              <string>Show Fitted Curve Crsp</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_7">
             <property name="text">
              <string>Show Hist Matched Crsp</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_8">
             <property name="text">
              <string>Show Crsp</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_10">
             <property name="text">
              <string>Show User Marked Crsp</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_11">
             <property name="text">
              <string>Show All Curves Points</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>R(%)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="SField_rad_doubleSpinBox">
               <property name="maximum">
                <double>1.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.010000000000000</double>
               </property>
               <property name="value">
                <double>0.040000000000000</double>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>w</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="SField_w_doubleSpinBox">
               <property name="maximum">
                <double>1.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.010000000000000</double>
               </property>
               <property name="value">
                <double>0.100000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>a</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="SField_a_doubleSpinBox">
               <property name="maximum">
                <double>10.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.010000000000000</double>
               </property>
               <property name="value">
                <double>0.700000000000000</double>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>b</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="SField_b_doubleSpinBox">
               <property name="maximum">
                <double>10.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.010000000000000</double>
               </property>
               <property name="value">
                <double>0.500000000000000</double>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_16">
               <property name="text">
                <string>c</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="SField_c_doubleSpinBox">
               <property name="minimum">
                <double>0.010000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.010000000000000</double>
               </property>
               <property name="value">
                <double>0.050000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Center</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="SField_WinCenter_Slider">
               <property name="maximum">
                <number>100</number>
               </property>
               <property name="value">
                <number>50</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Width</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="SField_WinWidth_Slider">
               <property name="maximum">
                <number>100</number>
               </property>
               <property name="value">
                <number>100</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_11">
             <item>
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>ARAP</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="lamd_ARAP_doubleSpinBox">
               <property name="decimals">
                <number>2</number>
               </property>
               <property name="singleStep">
                <double>0.010000000000000</double>
               </property>
               <property name="value">
                <double>0.100000000000000</double>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>flat</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="lamd_flat_doubleSpinBox">
               <property name="decimals">
                <number>2</number>
               </property>
               <property name="singleStep">
                <double>0.010000000000000</double>
               </property>
               <property name="value">
                <double>0.010000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>data</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="lamd_data_doubleSpinBox">
               <property name="decimals">
                <number>2</number>
               </property>
               <property name="singleStep">
                <double>0.010000000000000</double>
               </property>
               <property name="value">
                <double>0.500000000000000</double>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>SField</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="lamd_SField_doubleSpinBox">
               <property name="decimals">
                <number>2</number>
               </property>
               <property name="singleStep">
                <double>0.010000000000000</double>
               </property>
               <property name="value">
                <double>0.010000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_12">
             <item>
              <widget class="QLabel" name="label_14">
               <property name="text">
                <string>t1</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="source_curves_t1_doubleSpinBox">
               <property name="decimals">
                <number>2</number>
               </property>
               <property name="maximum">
                <double>1.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.010000000000000</double>
               </property>
               <property name="value">
                <double>0.750000000000000</double>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_15">
               <property name="text">
                <string>t2</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="target_curves_t2_doubleSpinBox">
               <property name="decimals">
                <number>2</number>
               </property>
               <property name="maximum">
                <double>1.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.010000000000000</double>
               </property>
               <property name="value">
                <double>0.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="ProjAlignViewGroupBox">
          <property name="title">
           <string>Projection Alignment Tools</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QCheckBox" name="Show_All_Lines_CheckBox">
             <property name="text">
              <string>Show All Constrained Lines</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="Show_ProjCrsp_CheckBox">
             <property name="text">
              <string>Show Projection Crsp</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Interactive Mode</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="Proj_Interact_Mode_ComboBox">
               <item>
                <property name="text">
                 <string>Draw Crsp Line</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Drag to Deform</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Correct Crsp</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Delete Target Curves</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Algorithm Tools</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QPushButton" name="NormalTransfer_PushButton">
             <property name="text">
              <string>Test Normal Transfer</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="NormalCompute_PushButton">
             <property name="text">
              <string>Do Normal Computing</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="Synthesis_PushButton">
             <property name="text">
              <string>Do Synthesis</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QPushButton" name="DetailSynthesis_PushButton">
               <property name="text">
                <string>LFReg</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="LFReg_NonRigid_PushButton">
               <property name="text">
                <string>NonRigid</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QSpinBox" name="LFReg_Method_SpinBox">
               <property name="value">
                <number>11</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="SField_Type_ComboBox">
               <property name="currentIndex">
                <number>1</number>
               </property>
               <item>
                <property name="text">
                 <string>Dist Map</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Tuned KD</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>ML's eq</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>415</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
       <zorder>FeatureViewGroupBox</zorder>
       <zorder>ProjAlignViewGroupBox</zorder>
       <zorder>groupBox</zorder>
       <zorder>groupBox_3</zorder>
       <zorder>groupBox_2</zorder>
      </widget>
      <widget class="QWidget" name="page_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>98</width>
         <height>28</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Page 2</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
